WestLaw UK.js

```diff
@@ -0,0 +1,139 @@
+{
+	"translatorID": "e6d9cf77-53e4-47f5-98ae-a69eba6d985f",
+	"label": "WestLaw UK",
+	"creator": "George Gebbett",
+	"target": "^https://uk\\.westlaw\\.com/Document/.*",
```

Comment:
We don't need the `.*` at the end - since the pattern starts with `^` (matches the beginning of the string) but doesn't end in `$` (matches the end of the string), we implicitly accept trailing characters not matched by the regex.

Reply:
Got rid of that